src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -47,6 +47,8 @@ public HashSet<int> Start(ApplicationData applicationData)
             LoadData();
             AddRows(win);
 
+            _filterField.Text = _applicationData.Filter;
```

Comment:
To make the code more robust, you should handle when Filter is null here instead of setting the default value to empty string in the other file for Filter.

You can do a simple:

```csharp
_applicationData.Filter ?? string.Empty
```

And then remove the default value in the cmdlet